ui/lib/sync/addon/components/secrets/page/destinations.hbs

```diff
@@ -53,7 +53,7 @@
     {{#each @destinations as |destination index|}}
       <ListItem
         @linkPrefix={{this.mountPoint}}
-        @linkParams={{array "secrets.destinations.destination.details" destination.type destination.name}}
+        @linkParams={{array "secrets.destinations.destination.secrets" destination.type destination.name}}
```

Comment:
Discussed with design that secrets should be the main landing page when clicking to a destination